MacOS/64bit/Sirius/sirius

```diff
@@ -158,6 +72,6 @@ fi
 
 
 "$JAVACMD" "$@" &
-wait
 
+wait
```

Comment:
That is different from the Linux version. I guess the call here should be same for both OS.